zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -167,19 +158,20 @@ impl SignVerifyConfig {
             challenges.keccak_input(),
         );
 
+        let output = meta.advice_column();
         // Ref. spec SignVerifyChip 1. Verify that keccak(pub_key_bytes) = pub_key_hash
         // by keccak table lookup, where pub_key_bytes is built from the pub_key
         // in the ecdsa_chip.
         let q_keccak = meta.complex_selector();
         meta.lookup_any("keccak", |meta| {
             // When address is 0, we disable the signature verification by using a dummy pk,
-            // msg_hash and signature which is not constrainted to match msg_hash_rlc nor
+            // msg_hash and signature which is not constrained to match msg_hash_rlc nor
             // the address.
             // Layout:
-            // | q_keccak |        a        |     rlc     |
-            // | -------- | --------------- | ----------- |
-            // |     1    | is_address_zero |    pk_rlc   |
-            // |          |                 | pk_hash_rlc |
+            // | q_keccak |        a        |     rlc     |    output   |
+            // | -------- | --------------- | ----------- | ----------- |
+            // |     1    | is_address_zero |    pk_rlc   |             |
+            // |          |                 |  pk_hash_lo | pk_hash_hi  |
```

Comment:
We can use `main_gate_config.advices()[1]` and `main_gate_config.advices()[2]` as word hi/lo, then we don't need to allocate a new advice column `output`.